akka-sample-persistence-dc-java/build.sbt

```
@@ -33,4 +33,7 @@ libraryDependencies ++= Seq(
   "org.scalatest" %% "scalatest" % "3.0.7" % "test"
 )
 
+// transitive dependency of akka 2.5x that is brought in by addons but evicted
+dependencyOverrides += "com.typesafe.akka" %% "akka-protobuf" % AkkaVersion
+
```

sbt couldn't find the proto for me:

```suggestion
// The default is src/main/protobuf, but the maven plugin expects them here:
sourceDirectory in ProtobufConfig := (sourceDirectory in Compile).value / "proto"

```

---

Ah thanks, i changed it to do the maven plugin's default and forgot about sbt!